### PR TITLE
updated 2016_ipdps_archer

### PR DESCRIPTION
**Pull request by Maryam81609**

### CODE REVIEW

File: _publications/2016_ipdps_agralslpm.md [hunk 1]
```diff
@@ -2,25 +2,24 @@
...
-shortname: 2016_IPDPS_Archer
+# The shortname is used for auto-generated titles
+shortname: IPDPS 2016
 # add a 2:1 aspect ratio (e.g., width: 400px, height: 200px) to the folder /assets/images/papers/
-image: 2016_IPDPS_Archer.png
-# 2018_tvcg_lineage.png
+image: 2016_ipdps_agralslpm.png
 # add a 2:1 aspect ratio teaser figure (e.g., width: 1200px, height: 600px) to the folder /assets/images/papers/
-image_large: 2016_IPDPS_Archer.png
+image_large: 2016_ipdps_agralslpm.png
```
Comment (zvonimir) [inline _publications/2016_ipdps_agralslpm.md:17]: You are missing the `_teaser` image, which should be 1200x600.

**Review by zvonimir — CHANGES_REQUESTED** [review]
Looks good, except that you are missing one image. Thx!

**Comment by Maryam81609** [comment]
I added the teaser image. I can test it tomorrow on my machine in the lab.

**Review by zvonimir — APPROVED** [review]
Looks good to me.

**Pull request merged**